muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/connect-field.tsx

```diff
@@ -257,12 +257,12 @@ export default class ConnectField extends React.Component<ConnectFieldProps, Con
         }
         return f;
       })
-    }, this.triggerChange)
+    }, triggerChange ? this.triggerChange : null)
```

Comment:
Shorter way to express this is could be:
```
triggerChange && this.triggerChange
```

Reply:
At least the naming of the variable should be different from the function so it would be easier to read in future.